### PR TITLE
Clehner/taggle integration

### PR DESCRIPTION
**Pull request by lehnerchristian**

### CODE REVIEW

File: src/lineup/internal/LineUpPanelActions.ts [hunk 1]
```diff
@@ -364,6 +364,18 @@ export default class LineUpPanelActions extends EventHandler {
         }
       });
   }
+
+  toggleDisableOverviewButton(disable: boolean = false) {
+    debugger;
```
Comment (sgratzl) [inline src/lineup/internal/LineUpPanelActions.ts]: why?

File: src/lineup/internal/LineUpPanelActions.ts [hunk 2]
```diff
@@ -364,6 +364,18 @@ export default class LineUpPanelActions extends EventHandler {
...
+    (<HTMLButtonElement>this.overview).disabled = disable;
+    if(disable) {
+      (<HTMLButtonElement>this.overview).title = `Overview disabled due to too many items in the table. Please filter the table below the threshold of ${MAX_AMOUNT_OF_ROWS_TO_DISABLE_OVERVIEW} items to enable the overview mode.`;
+      this.overview.style.cursor = 'not-allowed';
+    } else {
+      (<HTMLButtonElement>this.overview).title = `En/Disable Overview`;
+      this.overview.style.cursor = 'pointer';
```
Comment (sgratzl) [inline src/lineup/internal/LineUpPanelActions.ts]: isn't that the default?

File: src/lineup/internal/LineUpPanelActions.ts [hunk 3]
```diff
@@ -364,6 +364,18 @@ export default class LineUpPanelActions extends EventHandler {
         }
       });
   }
+
+  toggleDisableOverviewButton(disable: boolean = false) {
+    debugger;
+    (<HTMLButtonElement>this.overview).disabled = disable;
+    if(disable) {
+      (<HTMLButtonElement>this.overview).title = `Overview disabled due to too many items in the table. Please filter the table below the threshold of ${MAX_AMOUNT_OF_ROWS_TO_DISABLE_OVERVIEW} items to enable the overview mode.`;
```
Comment (sgratzl) [inline src/lineup/internal/LineUpPanelActions.ts]: cast not needed

File: src/lineup/internal/LineUpPanelActions.ts (hunk 1 above)
Comment (lehnerchristian) [reply src/lineup/internal/LineUpPanelActions.ts]: good question!

File: src/lineup/desc.ts [hunk 4]
```diff
@@ -165,7 +165,6 @@ export function createInitialRanking(provider: ADataProvider) {
   const ranking = provider.pushRanking();
   ranking.insert(provider.create(createAggregateDesc()), 0);
   ranking.push(provider.create(createSelectionDesc()));
-  ranking.push(provider.create(createGroupDesc()));
```
Comment (sgratzl) [inline src/lineup/desc.ts]: please remove the unused import, too

**Pull request merged**